src/structures/User.js

```diff
@@ -62,6 +62,12 @@ class User {
      * @type {?Message}
      */
     this.lastMessage = null;
+
+    /**
+     * Flag on the user. One of `STAFF`, `PARTNER`, `HYPESQUAD`
+     * @type {?String}
+     */
+    this.flag = data.flags ? Constants.UserFlags[data.flags] : null;
```

Comment:
So it's a complete coincidence that the numbers happen to be the first three possible flag values for a bitfield... and it's called `flags`, not `flag`...?